packages/connect/src/utils/firmwareReleaseConfigUtils.ts

```diff
@@ -6,122 +6,119 @@ import { getJWSPublicKey } from '@trezor/env-utils';
 import { firmwareReleaseConfigAssets } from './assetUtils';
 import { FirmwareUpdateSource, getOnlineFirmwareBaseUrl } from '../data/firmwareInfo';
 
-const JWS_SIGN_ALGORITHM = 'ES256';
-const VERSION = 1;
-const JWS_RELEASES_FILENAME_REMOTE = `releases.v${VERSION}.json`;
+const JWS_CONFIG = {
+    SIGN_ALGORITHM: 'ES256',
+    VERSION: 1,
+    REMOTE_FILENAME: 'releases.v1.json',
+    REQUEST_TIMEOUT_MS: 5000,
+};
+
+type JwsInfo = {
+    jws: string;
+    source: 'remote' | 'local';
+    env: FirmwareUpdateSource;
+};
 
-const getReleaseJWS = async () => {
+const fetchRemoteJWS = async (): Promise<JwsInfo> => {
```

Comment:
I'd be consistent with naming above and use:
```suggestion
const fetchRemoteJws = async (): Promise<JwsInfo> => {
```
Or maybe even:
```suggestion
const fetchRemoteJwsInfo = async (): Promise<JwsInfo> => {
```